pylds/tools.py

```diff
@@ -46,12 +46,12 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         LD = LD / LD.max()  # Scale LD output
     
     # Plot LDs
-    fig,ax = plt.subplots(1, 1, dpi = 100)
+    fig, (ax0, ax1) = plt.subplots(1, 2, figsize=(7.5,3), dpi=130)
```

Comment:
resolution can be defined as variables, as done before (See version in `master` branch, actually implemented by you @vkrajnak ), and perhaps figure canvas dimensions too

```
resolution = 130
fig_width = 7.5
fig_hight = 3
```

Reply:
Correction: it must have been implemented by you, the *VVesrion.ipynb (my major input for the master branch) doesn't have it.

I don't think adding a bunch of single use variables is good for readability. Note that this would make the subplots command a lot longer.